latex/epoch.tex

```diff
@@ -194,6 +192,40 @@ \subsection{Helper Functions}
   \label{fig:funcs:epoch-helper}
 \end{figure}
 
+
+The Figure~\ref{fig:defs:accounting} lists the accounting fields, denoted by $\Acnt$,
+which will be used throughout this section. It consists of:
+\begin{itemize}
+  \item The value $\var{treasury}$ tracks the amount of coin currently stored in the treasury.
+    Initially there will be no way to remove these funds.
+  \item The value $\var{reserves}$ tracks the amount of coin currently stored in the reserves.
+    This pot is used to pay rewards.
+  \item The value $\var{rewardPot}$ to tracks the rewards from the previous epoch that were not
+    payed out.
```

Comment:
```suggestion
    paid out.
```
"payed out" refers only to letting out slack in a rope, so probably not what you mean here :-)